salt/states/sysctl.py

```diff
@@ -79,16 +77,16 @@ def present(name, value, config=None):
                     "changed to {} in config file.".format(name, value)
                 )
                 return ret
-        elif name in configured and name not in current:
+        elif name in configured and current == "":
             ret["result"] = None
             ret["comment"] = (
                 "Sysctl value {0} is present in configuration file but is not "
                 "present in the running config. The value {0} is set to be "
                 "changed to {1}".format(name, value)
             )
             return ret
-        elif name in configured and name in current:
-            if str(value).split() == __salt__["sysctl.get"](name).split():
+        elif name in configured and current != "":
+            if str(value).split() == current.split():
```

Comment:
**question**: As I look at this `if/elif` chain would it be possible to rewrite this a bit clearer, perhaps by shuffling the order/indenting things?

Reply:
Ok.  I made the chain a little deeper but less long now.